src/Microsoft.AspNet.Authentication.OpenIdConnect/DefaultIdentityModelLoggingListener.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Diagnostics.Tracing;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Authentication.OpenIdConnect
+{
+    /// <summary>
+    /// Default logging event listener for wilson event source. It logs the data to the target where other Asp.Net logs go.
+    /// </summary>
+    public class DefaultIdentityModelLoggingListener : EventListener
```

Comment:
Yep, exactly.

The logging stack has .NET 4.5/portable targets that you could use for Wilson: https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Framework.Logging.Abstractions/project.json